src/setupWorker/glossary.ts

```diff
@@ -54,6 +59,14 @@ export type StartOptions = SharedOptions & {
    * instance is ready. Defaults to `true`.
    */
   waitUntilReady?: boolean
+
+  /**
+   * A predicate function that allows you override the default strict matching behavior.
+   * This is primarily useful in the scenario that you are using proxies you are using proxies
+   * and having difficulty getting the origin and scriptURL to match due to port differences.
+   * Default: (scriptURL, absoluteWorkerUrl) => worker.scriptURL ==== absoluteWorkerUrl
```

Comment:
Please do not include the default value in the JSDocs, that couples it too much to the implementation. Thanks.

Reply:
Noted.